File: Raven.Client.Lightweight/Connection/Profiling/IHoldProfilingInformation.cs
namespace Raven.Client.Connection.Profiling
{
	/// <summary>
	/// Interface for getting profiling information about the actions of the system
	/// within a given context, usually the context is database commands or async database commands
	/// </summary>
	public interface IHoldProfilingInformation
	{
		/// <summary>
		/// The profiling information
		/// </summary>
		ProfilingInformation ProfilingInformation { get; }
	}
}

File: Raven.Client.Java/src/main/java/net/ravendb/client/connection/profiling/IHoldProfilingInformation.java
package net.ravendb.client.connection.profiling;

/**
 * Interface for getting profiling information about the actions of the system
 * within a given context, usually the context is database commands or async database commands
 */
public interface IHoldProfilingInformation {
  /**
   * The profiling information
   */
  public ProfilingInformation getProfilingInformation();

  /**
   * Is expect 100 continue?
   * @return
   */
  public boolean isExpect100Continue();

}
